app/src/main/java/de/danoeh/antennapod/dialog/PlaybackControlsDialog.java

```diff
@@ -48,12 +48,11 @@ public void onStart() {
             public void loadMediaInfo() {
                 setupUi();
                 setupAudioTracks();
-                updateSpeed(new SpeedChangedEvent(getCurrentPlaybackSpeedMultiplier()));
             }
         };
         controller.init();
         setupUi();
-        EventBus.getDefault().register(this);
+        //EventBus.getDefault().register(this);
```

Comment:
Both registering and unregistering should be removed. Please remove the code, not just comment it out.